assets/vega_lite/main.js

```diff
@@ -129,7 +129,12 @@ export function init(ctx, data) {
   let theme = config.theme === "livebook" ? livebookTheme : {};
 
   const options = {
-    actions: { export: true, source: false, compiled: false, editor: false },
```

Comment:
Actually, I think we can enable all of these.

I am almost sure we disabled the other ones, because they open a new tab and it didn't work from within iframe in the past, but we changed the iframe permissions and it works now.

Reply:
compiled source is not that helpful, imho. The others are useful only if you want to debug/inspect the underlying vega-lite representation. Your call though.

Reply:
Without compiled sounds good to me. It's a menu anyway, so I think always having the extra options doesn't hurt :)

Reply:
Updated

Reply:
Ah, what I meant was that I wouldn't make these configurable altogether and just enable these actions. Unless you have a use case where you'd like to disable the options :)

Reply:
Done